src/main/java/io/jenkins/update_center/HealthScores.java

```diff
@@ -0,0 +1,62 @@
+package io.jenkins.update_center;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+import com.alibaba.fastjson.JSON;
+import io.jenkins.update_center.util.HttpHelper;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+
+public class HealthScores {
```

Comment:
Javadoc for this would be useful.

Reply:
Just added what made sense to me. Please let me know if it's alright or if I need to improve it.